tests/p2p/test_split_brain.py

```diff
@@ -328,13 +321,11 @@ def test_split_brain_only_blocks_bigger_score(self):
         # Assert that the consensus had the manager2 chain
         self.assertEqual(winners2_blocks, manager2_blocks)
 
-    def test_split_brain_no_double_spending(self):
+    def test_split_brain_no_double_spending(self) -> None:
         manager1 = self.create_peer(self.network, unlock_wallet=True)
-        manager1.avg_time_between_blocks = 3
         manager1.connections.disable_rate_limiter()
 
         manager2 = self.create_peer(self.network, unlock_wallet=True)
-        manager2.avg_time_between_blocks = 3
```

Comment:
Why are you changing this?

Reply:
After this test was typed, mypy complained that `HathorManager` does not have a `avg_time_between_blocks` property. Therefore, this was useless (probably an artifact from some older code) and removing it doesn't break the test.

Reply:
This line was commented instead. So, we are going to fix it later.